packages/fxa-settings/src/components/Settings/DropDownAvatarMenu/index.tsx

```diff
@@ -64,7 +64,7 @@ export const DropDownAvatarMenu = () => {
           aria-label={dropDownMenuTitle}
           aria-expanded={!!isRevealed}
           aria-haspopup="menu"
-          className="rounded-full border border-transparent hover:border-purple-500 focus:border-purple-500 focus:outline-none active:border-purple-700 transition-standard"
+          className="rounded-full border border-transparent hover:border-purple-500 focus-visible:outline focus-visible:outline-blue-500 focus-visible:outline-2 active:border-purple-700 transition-standard"
```

Comment:
The focus outline looks great 👌 Do you have ideas about how to update the hover styling which is still purple and conflicts a tad with the focus outline?